specification/desktopvirtualization/resource-manager/readme.ruby.md

```diff
@@ -12,15 +12,16 @@ azure-arm: true
 
 ``` yaml $(ruby) && $(multiapi)
 batch:
-  - tag: package-2019-01-23-preview
+  - tag: package-2019-12-10-preview

```

Comment:
ruby is always a multi-api sdk structure. Since you have three preview versions in the swagger spec, I recommend having three tags under this `batch:`. Meanwhile, keeping three tag definitions in this readme.ruby.md as well. It's fine to have only one tag in ruby sdk if you insist on doing this.

Reply:
ok adding the tags to the batch.